.github/workflows/setup_pwa_manual.yml

```diff
@@ -0,0 +1,137 @@
+name: SalesforceCommerceCloud/pwa-kit/e2e
+on:
+  workflow_dispatch:
+    inputs:
+      use_extensibility:
+        type: boolean
+        description: Use Extensibility?
+        default: true
```

Comment:
Why do we need this? isn't extensibility determined based on the code in the branch this workflow is using and not an option on the workflow itself?

Reply:
Just added all the parameters that we have for the command